site/plugins/cdn/index.php

```diff
@@ -41,7 +41,7 @@
 			static $original;
 
 			if ($kirby->option('cdn', false) !== false) {
-				$url = cdn($file, $options);
+				$url = Image::url($file, $options);
```

Comment:
`Suggestion:` We could move this into `Kirby\Cdn\Fileversion` making sure that its URL always uses `Kirby\Cdn\Image::url()`.

Reply:
You mean the URL generation? Would mean that the constructor doesn't get the `url` prop but instead determines it from the original and modifications by calling `Image::url()`, right?

Reply:
Yes, that was my thought. Like

```php
	public function __construct(array $props)
	{
		parent::__construct([
			...$props,
			'url' => Image::url($props['original'], $props['modifications'])
		]);
	}
```